e2e-tests/cypress/e2e/journeys/insurance/your-business/company-details/company-details-save-and-back.spec.js

```diff
@@ -0,0 +1,107 @@
+import { companyDetails } from '../../../../pages/your-business';
+import { saveAndBackButton, yesRadioInput } from '../../../../pages/shared';
+import {
+  ROUTES, FIELD_IDS, INVALID_PHONE_NUMBERS, WEBSITE_EXAMPLES, COMPANIES_HOUSE_NUMBER, VALID_PHONE_NUMBERS,
+} from '../../../../../../constants';
+import getReferenceNumber from '../../../../helpers/get-reference-number';
+
+const { ALL_SECTIONS } = ROUTES.INSURANCE;
+
+const {
+  EXPORTER_BUSINESS: {
+    COMPANY_HOUSE: {
+      INPUT,
+    },
+    YOUR_COMPANY: {
+      TRADING_NAME,
+      TRADING_ADDRESS,
+      WEBSITE,
+      PHONE_NUMBER,
+    },
+  },
+} = FIELD_IDS.INSURANCE;
+
+describe("Insurance - Your business - Company details page - As an Exporter I want to enter details about my business in 'your business' section", () => {
```

Comment:
What do you think about having a simplified version for "save and back"? I've gone with this - happy to change these though

```js
context('Insurance - Policy and exports - About goods or services - Save and go back', () => {
```

Either way definitely think we need to have "save and go back" in the description